src/main/java/edu/greenblitz/tobyDetermined/commands/TelescopicArm/Commands/armUtils.java

```diff
@@ -0,0 +1,9 @@
+package edu.greenblitz.tobyDetermined.commands.TelescopicArm.Commands;
+
+import edu.greenblitz.tobyDetermined.subsystems.telescopicArm.Elbow;
+
+public class armUtils {
+    public static boolean isInTheSameSide (double angle1, double angle2) {//in rads
```

Comment:
This function checks if they are on the same side, and calls a function that checks that they are in the same state. (Rename one of the functions)